src/login/LoginMain/LoginMain_actions.js

```diff
@@ -0,0 +1,25 @@
+//export const APP_FETCHING = "APP_FETCHING";
+export const APP_LOADED = "APP_LOADED";
+export const NEW_CSRF_TOKEN = "NEW_CSRF_TOKEN";
+
+
+export function appLoaded() {
+  return {
+    type: APP_LOADED
+  };
+}
+
+export function newCsrfToken(token) {
+  return {
+    type: NEW_CSRF_TOKEN,
+    payload: {
+      csrf_token: token
+    }
+  };
+}
```

Comment:
Would it be possible to start logging in comments what these actions do? I would be nice to know what happens and what prop it controls in the component without having to look in the reducer...maybe something like the below (?): 
> // controls 'is_app_loaded' (boolean), which renders app once the config data is received (or whatever is accurate)

Maybe this wont work in the log run, but id like to try